AppCenter/AppCenter/Internals/Storage/MSLogDBStorage.m

```diff
@@ -12,14 +18,17 @@ - (instancetype)init {
     kMSLogTableName : @[
```

Comment:
Could be good to have a warning here saying don't modify the schema without a migration plan and bumping the version.